firstvoices/backend/resources/dictionary.py

```diff
@@ -21,14 +23,17 @@
     TypeOfDictionaryEntry,
 )
 from backend.resources.base import (
+    AudienceMixin,
     BaseResource,
     RelatedMediaResourceMixin,
     SiteContentResource,
 )
 from backend.resources.utils.import_export_widgets import ChoicesWidget
 
 
-class DictionaryEntryResource(SiteContentResource, RelatedMediaResourceMixin):
+class DictionaryEntryResource(
+    AudienceMixin, SiteContentResource, RelatedMediaResourceMixin
```

Comment:
You can use ControlledSiteContentResource instead to get the standard visibility field